src/plugins/plot/chart/MctChart.vue

```diff
@@ -252,12 +251,19 @@ export default {
     this.listenTo(this.config.xAxis, 'change:displayRange', this.scheduleDraw);
     this.listenTo(this.config.xAxis, 'change', this.redrawIfNotAlreadyHandled);
     this.config.series.forEach(this.onSeriesAdd, this);
-    this.$emit('chart-loaded');
+    this.$emit('chartLoaded');
+
+    this.handleWindowResize = _.debounce(this.handleWindowResize, 500);
```

Comment:
I think we can cut this debounce time by at least half to make the limit line re-render feel more seamless. There's a noticeable delay with the 500ms value

Reply:
Typically we use 500 ms. But I can reduce it to 300